tools/ubuntu-setup/java8.sh

```diff
@@ -2,8 +2,10 @@
 set -e
 set -x
 
-sudo apt-get install -y software-properties-common
-sudo add-apt-repository -y ppa:webupd8team/java
-sudo apt-get update -y
-echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | sudo /usr/bin/debconf-set-selections
-sudo apt-get install -y oracle-java8-installer
+if [ "$(lsb_release -cs)" == "trusty" ]; then
```

Comment:
Nice to have this consistency of openjdk8 for both ubuntu 14 or above

Reply:
Just hoping 'jonathonf' keeps his PPA up-to-date until trusty goes EOL.